### PR TITLE
Update chain.json

### PR DESCRIPTION
**Pull request by reversesigh**
Desmos
Adding 4.8.1 patch - https://github.com/desmos-labs/desmos/releases/tag/v4.8.1

Sommelier
Update from v5 to v6 at 8704480 pending approval of Prop 47: https://www.mintscan.io/sommelier/proposals/47

XPLA
Adding seed, RPC and REST for Lavender.Five Nodes

### CODE REVIEW

File: sommelier/chain.json [hunk 1]
```diff
@@ -29,9 +29,9 @@
   },
   "codebase": {
     "git_repo": "https://github.com/PeggyJV/sommelier",
-    "recommended_version": "v5.0.0",
```
Comment (JeremyParish69) [inline sommelier/chain.json:32]: Can we please also create a new object at the bottom of the versions array for this v6?

File: desmos/chain.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
         "name": "v4.8.0",
         "recommended_version": "v4.8.0",
         "compatible_versions": [
-          "v4.8.0"
+          "v4.8.0", "v4.8.1"
```
[the remark this answers is not in the document]
Comment (reversesigh) [reply desmos/chain.json:54]: This should be corrected now and has been committed. Thanks for pointing this out.

File: sommelier/chain.json (hunk 1 above)
Comment (reversesigh) [reply sommelier/chain.json:32]: Apologies for overlooking that. Resubmitting with proper changes now.

Comment (JeremyParish69) [reply sommelier/chain.json:32]: looks like it's still removing v5 even though we want to keep that

Comment (reversesigh) [reply sommelier/chain.json:32]: Apologies if I'm not following- if the current chain version is v6, why do we keep v5 on here?

Comment (JeremyParish69) [reply sommelier/chain.json:32]: to keep a record of versions. Ideally we have all versions in here, with their effective block height.

**Pull request merged**